tests/subyt/templates/02-collection.ttl

```diff
@@ -0,0 +1,38 @@
+{# Template '02-collection.ttl' 
+  Description: 'This template shows how to create and apply a translation map from one of the input sets; 
+                and does so in classic iteration mode.' 
+  Author: Marc Portier
+  Date: 08/03/2022
+  Target vocabs: 
+    - ex: <https://example.org/02-collection/>
+  Inputs: 
+    - input-file: (None)
+    - set-file(s): 
+        - data_countries.csv as countries,
+        - data.csv as _
+    - mode: iteration,
+#}
+
+{% if ctrl.isFirst %}
+@prefix ex: <https://example.org/02-collection/> .
+ex:fullset ex:iteration {{ctrl.settings.iteration | ttl("xsd:boolean")}};
+    ex:ctrl-first {{ctrl.isFirst | ttl("xsd:boolean")}};
```

Comment:
replace the historic `| ttl` usage with the new style `| xsd`

again grep around in the samples to get rid of all occurances